client/reader/stream/index.jsx

```diff
@@ -63,52 +57,45 @@ const HEADER_OFFSET_TOP = 46;
 const noop = () => {};
 const pagesByKey = new Map();
 const inputTags = [ 'INPUT', 'SELECT', 'TEXTAREA' ];
-const excludesSidebar = [
-	'a8c',
-	'conversations',
-	'conversations-a8c',
-	'feed',
-	'likes',
-	'search',
-	'list',
-	'p2',
-];
 
 class ReaderStream extends Component {
 	static propTypes = {
-		translate: PropTypes.func,
-		trackScrollPage: PropTypes.func.isRequired,
-		suppressSiteNameLink: PropTypes.bool,
-		showPostHeader: PropTypes.bool,
-		showFollowInHeader: PropTypes.bool,
-		onUpdatesShown: PropTypes.func,
-		emptyContent: PropTypes.object,
 		className: PropTypes.string,
-		showDefaultEmptyContentIfMissing: PropTypes.bool,
-		placeholderFactory: PropTypes.func,
+		emptyContent: PropTypes.object,
 		followSource: PropTypes.string,
+		forcePlaceholders: PropTypes.bool,
+		intro: PropTypes.object,
 		isDiscoverStream: PropTypes.bool,
-		useCompactCards: PropTypes.bool,
 		isMain: PropTypes.bool,
-		intro: PropTypes.object,
-		forcePlaceholders: PropTypes.bool,
+		onUpdatesShown: PropTypes.func,
+		placeholderFactory: PropTypes.func,
 		recsStreamKey: PropTypes.string,
+		showDefaultEmptyContentIfMissing: PropTypes.bool,
 		showFollowButton: PropTypes.bool,
+		showFollowInHeader: PropTypes.bool,
+		showPostHeader: PropTypes.bool,
+		sidebarTabTitle: PropTypes.string,
+		streamHeader: PropTypes.element,
+		streamSidebar: PropTypes.element,
+		suppressSiteNameLink: PropTypes.bool,
+		trackScrollPage: PropTypes.func.isRequired,
+		translate: PropTypes.func,
+		useCompactCards: PropTypes.bool,
```

Comment:
This alphabetizes and adds 3 missing props: `streamSidebar`, `streamHeader`, and `sidebarTabTitle`.